packages/core/src/rest/ZosmfRestClient.ts

```diff
@@ -20,6 +20,7 @@ import { ZosmfHeaders } from "./ZosmfHeaders";
  * @extends {RestClient}
  */
 export class ZosmfRestClient extends RestClient {
+    mDecode: boolean;
```

Comment:
I'm a little confused as to why this is needed in this PR.
@t1m0thyj, was this updated in master?

Reply:
Yes, `mDecode` should already be defined on master branch:
![image](https://user-images.githubusercontent.com/22344007/111162601-acfd9d00-8572-11eb-84fa-1115a3299758.png)

I encountered a similar error this morning where CLI failed to build because of complaints about `mDecode`, deleting all package-lock.json files and reinstalling fixed it:
```bash
cd zowe-cli
rm -f package-lock.json
rm -f packages/**/package-lock.json
npm install
```